src/test/java/io/castle/client/CastleIdentifyHttpTest.java

```diff
@@ -0,0 +1,103 @@
+package io.castle.client;
+
+import io.castle.client.model.AuthenticateAction;
+import io.castle.client.model.AuthenticateFailoverStrategy;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class CastleIdentifyHttpTest extends AbstractCastleHttpLayerTest {
+
+    public CastleIdentifyHttpTest() {
+        super(new AuthenticateFailoverStrategy(AuthenticateAction.CHALLENGE));
+    }
+
+    @Test
+    public void identifyEndpointTest() throws InterruptedException {
+        //given
+        server.enqueue(new MockResponse().setResponseCode(200));
+        String id = "12345";
+
+        // and a mock Request
+        HttpServletRequest request = new MockHttpServletRequest();
+
+        // when and an identify request is made
+        sdk.onRequest(request).identify(id);
+
+        // then the json match specification
+        RecordedRequest recordedRequest = server.takeRequest();
+        Assert.assertEquals("{\"user_id\":\"12345\",\"active\":true,\"context\":{\"active\":true,\"ip\":\"127.0.0.1\",\"headers\":{\"REMOTE_ADDR\":\"127.0.0.1\"},\"library\":{\"name\":\"Castle\",\"version\":\"0.6.0-SNAPSHOT\"}}}",
```

Comment:
@paweld2 just noticed this: active flag should *only* be part of context not again as first level key in the payload. Just noticed here, can you please double check if it occurs somewhere else?

Reply:
Done.
We fix the identify method and check that all the endpoints set the value on context.active field.